concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1363,9 +1363,9 @@ jobs:
                   echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
 
                   if cf service-brokers | grep "rds-broker\s"; then
-                    cf update-service-broker rds-broker rds-broker $RDS_BROKER_PASS http://$RDS_BROKER_SERVER
+                    cf update-service-broker rds-broker rds-broker $RDS_BROKER_PASS https://$RDS_BROKER_SERVER
```

Comment:
This is wrong, because these commands will run AFTER you remove the HTTP listener! so we will have downtime because the broker is registered with the wrong old url for a while.